test/e2e/egress_firewall.go

```diff
@@ -637,6 +459,195 @@ spec:
     to:
       cidrSelector: %s
 `, f.Namespace.Name, exFWPermitTcpDnsDest, singleIPMask, exFWPermitCIDR, exFWDenyCIDR)
-		applyEF(egressFirewallConfig, f.Namespace.Name)
-	})
+				applyEF(egressFirewallConfig, f.Namespace.Name)
+				framework.Logf("Deleting EgressFirewall in namespace %s", f.Namespace.Name)
+				e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "delete", "egressfirewall", "default")
+			}
+
+			srcPodName := "e2e-egress-fw-src-pod"
+			testContainer := fmt.Sprintf("%s-container", srcPodName)
+			testContainerFlag := fmt.Sprintf("--container=%s", testContainer)
+			// use random labels in case test runs again since it's a pain to remove the label from the node
```

Comment:
why is it a pain? if everyone does this we could end up with flakes in the future.

Reply:
lol, you tell me https://github.com/ovn-org/ovn-kubernetes/commit/34350bde6f5a56904cc0b81751bfc0e0a6a7684f#diff-65b73a6a62a4e5b8903b9f17f0e03a42365b127e7d86f006f9bec363b4be04b8R1276
I just moved that part :D

Reply:
LOL! I vaguely remember writing that and had originally written code to remove labels...but i dont remember why I ended up doing it. Since you are just moving code we can ignore it for now.